x-pack/plugins/cloud_security_posture/public/common/api/use_compliance_dashboard_data_api.ts

```diff
@@ -7,14 +7,36 @@
 
 import { type QueryObserverOptions, useQuery } from '@tanstack/react-query';
 import { useKibana } from '../hooks/use_kibana';
-import { ComplianceDashboardData } from '../../../common/types';
+import { PosturePolicyTemplate, ComplianceDashboardData } from '../../../common/types';
 import { STATS_ROUTE_PATH } from '../../../common/constants';
 
-const getStatsKey = ['csp_dashboard_stats'];
+const getCspmStatsKey = ['csp_cspm_dashboard_stats'];
+const getKspmStatsKey = ['csp_kspm_dashboard_stats'];
 
-export const useComplianceDashboardDataApi = (
+export const getStatsRoute = (policyTemplate: PosturePolicyTemplate) => {
+  return STATS_ROUTE_PATH.replace('{policy_template}', policyTemplate);
+};
+
+export const useCspmComplianceDashboardDataApi = (
+  options: QueryObserverOptions<unknown, unknown, ComplianceDashboardData, unknown, string[]>
+) => {
+  const { http } = useKibana().services;
+  return useQuery(
+    getCspmStatsKey,
+    // TODO: CIS AWS - remove casting and use actual policy template instead of benchmark_id
+    () => http.get<ComplianceDashboardData>(getStatsRoute('cis_aws' as PosturePolicyTemplate)),
+    options
```

Comment:
consider changing `getStatsRoute` to take a `PostureInput` so we don't need to have this comment +casting twice, then just change that function.

Reply:
as commented above, while generally true, it will have to be reverted in the near future if I'll add this change now, for now, we decided to keep the temporary code